dicee/config.py

```diff
@@ -152,6 +152,13 @@ def __init__(self, **kwargs):
 
         self.n_epochs_eval_model: str = "val_test"
         """Evaluating link prediction performance on data splits while performing periodic evaluation."""
+        
+        self.adaptive_lr = dict()
+        """Adaptive learning rate parameters, e.g., {"lr_decay": 0.1}"""
+
+        self.swa_start_epoch: int = 1
```

Comment:
Please initialize the swa_start_epoch as None so that the config does not confuses people